juju/model.py

```diff
@@ -1266,7 +1264,9 @@ def _get_series(self, entity_url, entity):
             '24/lxd/3' - place in container 3 on machine 24
 
             If None, a new machine is provisioned.
-
+        :param bool trust: Trust signifies that the charm should be deployed 
+            with access to trusted credentials. Hooks run by the charm can access
+            cloud credentials and other trusted access credentials.
```

Comment:
worth it to drop a link to the docs here for it?

Reply:
I'm unsure, as we can't link to a heading in this atm https://docs.jujucharms.com/deploying-advanced-applications

Reply:
Ah, missed it up there ty